common/lib/common-utils/src/disposal.ts

```diff
@@ -10,6 +10,8 @@ import { IDisposable } from "@fluidframework/common-definitions";
  * {@link @fluidframework/common-definitions#IDisposable | disposable} object has not yet been disposed.
  *
  * @throws Will throw an error if the item has already been disposed.
+ *
+ * @deprecated Moved to the `@fluidframework/container-loader` package.
```

Comment:
Nit: since it's not exported in container-loader, not sure that bit serves much purpose as part of the deprecation message. If it's for our reference, I'd rely on either the PR or maybe a `@privateremarks` TSDoc section. Similar comment for some of the other deprecation messages.

Reply:
Good point; I'll update this comment.